src/types/IWLedState.ts

```diff
@@ -7,5 +7,5 @@ import IWledSegment from "./IWledSegment";
 export default interface IWledState {
   on?: boolean;
   effect?: number;
-  seg?: IWledSegment[];
+  seg?: IWledSegment | IWledSegment[];
```

Comment:
Is there any reason to keep the array version? I never really supported segments with this in the first place and was only trying to get it far enough along that all lights would change. Given what you've said that is simply sending a segment object, not an array.

Reply:
Wait, never mind. This is fine. The object model correctly reflects what the API can support, not what this node uses. I need more coffee..